resources/install/postinst

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+python3 -m venv /usr/bin/veles-server/venv
```

Comment:
Hmm, I don't know if `/usr/bin/` is the best place for this venv, but I don't know where should we put it :(